src/containers/repository.py

```diff
@@ -21,3 +46,201 @@ def load_from_json(self, repository):
         self.organization_name = repository["orgName"]
         self.repository_name = repository["repoName"]
         self.query_labels = repository["queryLabels"]
+
+    def get_gh_projects(self, session: requests.sessions.Session) -> List[GHProject]:
```

Comment:
This type of method should be here. They should be moved to mention GithubManager class.

Reply:
Method was deleted during refactoring phase.